udev-utils.c

```diff
@@ -116,13 +145,36 @@ get_subsystem_config_by_syspath(const char *path)
 	return (NULL);
 }
 
+static bool
+kernel_has_evdev_enabled()
+{
+	static int enabled = -1;
+	size_t len;
+
+	if (enabled != -1)
+		return (enabled);
+
+	if (sysctlbyname("kern.features.evdev", &enabled, &len, NULL, 0) < 0)
```

Comment:
I think sysctl's name should be kern.features.evdev_support
kern.features.evdev sysctl appears if evdev.ko is kld-loaded or compiled-in.